sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md

```diff
@@ -28,6 +28,66 @@ For highly sensitive data, clients should consider additional layers of protecti
 Key Vault also supports a contentType field for secrets. Clients may specify the content type of a secret to assist in interpreting the secret data when it's retrieved. The maximum length of this field is 255 characters. There are no pre-defined values. The suggested usage is as a hint for interpreting the secret data.
 
 Besides, this starter provides features of supporting multiple Key Vaults, case sensitive mode of Key Vault names and using placeholder presenting Key Vault names in property file
+
+### Configuration Options
+Azure Spring Boot Key Vault Starter deprecates all legacy properties of which the prefix is `azure.keyvault` and uses `spring.cloud.azure.keyvault` instead.
```

Comment:
tagging @vcolin7 for more feedback

Reply:
We should keep the name consistent across the README.
```suggestion
Azure Key Vault Secrets Spring Boot Starter deprecates all legacy properties of which the prefix is `azure.keyvault` and uses `spring.cloud.azure.keyvault` instead.
```